boefjes/boefjes/plugins/kat_answer_parser/normalize.py

```diff
@@ -0,0 +1,18 @@
+import logging
+from typing import Iterable, Union
+
+from boefjes.job_models import NormalizerMeta
+from octopoes.models import OOI
+from octopoes.models.ooi.config import Config
+
+logger = logging.getLogger(__name__)
+
+
+def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterable[OOI]:
+    if "/bit/port-classification" in [mime_type["value"] for mime_type in normalizer_meta.raw_data.mime_types]:
```

Comment:
Personally, I found it hard to read (`if ... in <listcompr>:`. Can you consider extracting this list comprehension to a variable? Also, in this case, you're better off with a generator comprehension instead of a list comprehension.

I'll leave it up to you.